openhands/server/session/agent.py

```diff
@@ -105,16 +132,31 @@ async def _create_controller(
         agent_to_llm_config: dict[str, LLMConfig] | None = None,
         agent_configs: dict[str, AgentConfig] | None = None,
     ):
-        """Creates an AgentController instance."""
+        """Creates an AgentController instance
+
+        Parameters:
+        - agent (Agent):
+        - confirmation_mode (bool): Whether to use confirmation mode
+        - max_iterations (int):
+        - max_budget_per_task (float):
+        - agent_to_llm_config (dict[str, LLMConfig]):
+        - agent_configs (dict[str, AgentConfig]):
+        """
+
         if self.controller is not None:
             raise RuntimeError('Controller already created')
         if self.runtime is None:
             raise RuntimeError(
                 'Runtime must be initialized before the agent controller'
             )
 
-        logger.info(f'Agents: {agent_configs}')
-        logger.info(f'Creating agent {agent.name} using LLM {agent.llm.config.model}')
+        logger.info(
+            '\n--------------------------------- OpenHands Configuration ---------------------------------\n'
+            f'LLM: {agent.llm.config.model}\n'
+            f'Base URL: {agent.llm.config.base_url}\n'
+            f'Agent: {agent.name}\n'
+            '-------------------------------------------------------------------------------------------'
+        )
```

Comment:
The "-" may be excessive but the log is too busy and I believe this is one of the most important things we log so it's important that it's visible.

Reply:
It's fine for now since it adds a new line by itself. Looks pretty clean